astro/package.json

```
@@ -9,7 +9,9 @@
 		"dev-build": "DEV=true astro build",
 		"postbuild": "pagefind",
 		"preview": "astro preview",
-		"astro": "astro"
+		"astro": "astro",
+		"vale": "./node_modules/@vvago/vale/bin/vale",
+		"spellcheck": "npm run vale src/content"
```

```suggestion
		"spellcheck": "npm run vale src/content src/components"
```

Want to spell check both of these areas, since they both create user facing content.